src/style.css

```diff
@@ -1,4 +1,62 @@
-.hello {
-  color: red;
-  background: url('./img.png');
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif;
+  color: #545862;
+  font-weight: 400;
+  border: none;
+  font-size: calc(1.5vh + 1px);
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #545862;
+}
+
+a {
```

Comment:
@Shedrack-Sunday I has the border. If you check my css it has the same border style as in the original design. I got the border shadow from the original website. It would have been great if the other reviewers told me about these changes before because now I have to use an extra review for these small changes.

Reply:
@Shedrack-Sunday I have changes the body background. Now the border shadow should be visible

Reply:
Sorry for the inconvenience, It seems you didn't understand them fully. one of the reveivwers said "Kindly make sure to fix your design so it looks exactly like the given design, as for now " then went on to give an image comparison.